src/documents/InfusionEventSystem.md

```diff
@@ -46,11 +46,13 @@ A top-level options block named **`events`** is supported on every component der
   </tr>
 </table>
 
+**NOTE**: _`preventable` events are very rarely used and will soon be deprecated in the framework_.
```

Comment:
I guess I can't firmly predict that they will indeed be deprecated - although I'm not particularly looking forward to a community discussion on whether they should be deprecated or not. I added this sentence because I noticed that in the first draft of his Infusionised components, @the-t-in-rtf had made each of his events "preventable" by default and I wanted to highlight in the docs that this represented at the very least an unusual implementation choice. I used to think that there was no valid use case for an event which terminated processing, since we had no detailed control over the positioning of event listeners in a chain - but with the introduction of priority constraints, this is no longer true. You could now consider a "preventable" event is one that operates a semantic similar to the kinds of "transforming promise chain pseudoevents" that are starting to appear, where a rejection is received in the middle of the chain. I think that with the introduction of promise idioms into the framework, we will start to have other kinds of events built into the core... such as "enlatening events" to handle the use case that @the-t-in-rtf  ran into with his pouchdb wrapper, where he has a piece of on-destruction cleanup that he needs to perform that is asynchronous - and that we should not advertise that the component is destroyed until it is done (that is, "afterDestroy" could be delayed to an arbitrary point following "onDestroy" rather than occuring synchronously after it). 

I could either say "may soon be deprecated" or as we have in other places, "talk to us if you want to try to use these".